lib/spack/spack/cray_manifest.py

```diff
@@ -46,9 +47,29 @@ def translated_compiler_name(manifest_compiler_name):
         )
 
 
-def compiler_from_entry(entry):
+def compiler_from_entry(entry, manifest_path):
+    # Note that manifest_path is only passed here to compose a
+    # useful warning message paths appear to be missing.
```

Comment:
I think there is a typo:
```suggestion
    # Note that manifest_path is only passed here to compose a
    # useful warning message when paths appear to be missing.
```